src/style.css

```
@@ -0,0 +1,49 @@
+:root {
+  --stars-margin: 4px;
+  --stars-width: 16px;
+  --stars-height: 16px;
```

Since star is a square, those values are the same
```suggestion
  --star-dimensions 16px;
```